adapter/config/parser.go

```diff
@@ -162,3 +164,46 @@ func GetControlPlaneConnectedTenantDomain() string {
 	}
 	return superTenantDomain
 }
+
+func (config *Config) resolveDeprecatedProperties() {
+	if config.ControlPlane.ServiceURLDeprecated != UnassignedAsDeprecated {
+		printDeprecatedWarningLog("controlPlane.serviceUrl", "controlPlane.serviceURL")
+		config.ControlPlane.ServiceURL = config.ControlPlane.ServiceURLDeprecated
+	}
+	if config.GlobalAdapter.ServiceURLDeprecated != UnassignedAsDeprecated {
+		printDeprecatedWarningLog("globalAdapter.serviceUrl", "globalAdapter.serviceURL")
+		config.GlobalAdapter.ServiceURL = config.GlobalAdapter.ServiceURLDeprecated
+	}
+	if config.Enforcer.Throttling.JmsConnectionProviderURLDeprecated != UnassignedAsDeprecated {
+		printDeprecatedWarningLog("enforcer.throttling.JmsConnectionProviderUrl", "enforcer.throttling.JmsConnectionProviderURL")
+		config.Enforcer.Throttling.JmsConnectionProviderURL = config.Enforcer.Throttling.JmsConnectionProviderURLDeprecated
+	}
+	if config.GlobalAdapter.OverwriteHostName != UnassignedAsDeprecated {
+		printDeprecatedWarningLog("globalAdapter.OverwriteHostName", "globalAdapter.OverrideHostName")
```

Comment:
shouldn't we override the value here?
```suggestion
		printDeprecatedWarningLog("globalAdapter.OverwriteHostName", "globalAdapter.OverrideHostName")
		onfig.GlobalAdapter.OverrideHostName = onfig.GlobalAdapter.OverwriteHostName
```

Reply:
fixed.